src/components/AssetDescriptionComponent.jsx

```diff
@@ -98,13 +101,17 @@ class AssetDescriptionComponent extends React.Component {
     const {
       assetAsigneeUsers,
       assignedUser,
-      toggleModal,
-      buttonLoading,
       assetDetail,
-      errorMessage
+      errorMessage,
+      toggleModal,
+      buttonLoading
     } = this.props;
+
     const triggerProps = this.triggerProps();
 
+    const showAssignDropdown =
+      assetDetail.current_status === 'Available' || assetDetail.current_status === 'Allocated';
```

Comment:
Maybe we can create constants for `Available` and `Allocated`

Reply:
Sure thing